src/gui/systray.cpp

```diff
@@ -57,6 +58,8 @@ void Systray::setTrayEngine(QQmlApplicationEngine *trayEngine)
 
     _trayEngine->addImportPath("qrc:/qml/theme");
     _trayEngine->addImageProvider("avatars", new ImageProvider);
+    // TODO: Better name
+    _trayEngine->addImageProvider(QLatin1String("colorsvgimages"), new ColorSvgImageProvider);
```

Comment:
I'd strive to have it support the raster case too. There are still branded customers using png for instance and it will be the case for some time AFAIK.

In which case for the naming I'd go for: "palettecolorizedimages" and "PaletteColorizedImageProvider". The important thing being that a) it colorized and b) it does so respecting the system palette. "svg" is more of an implementation detail (which see above I argue it shouldn't be limited to svg anyway).